draco/util/kernels.py

```diff
@@ -1,8 +1,7 @@
 """Routines for creating kernel/covariance matrices."""
```

Comment:
This diff is also a bit painful. I haven't actually removed much, just reshuffled a bit